Content.Shared/Morgue/SharedMorgue.cs

```diff
@@ -1,19 +1,18 @@
-﻿using Robust.Shared.Serialization;
+using Robust.Shared.Serialization;
 
-namespace Content.Shared.Morgue
+namespace Content.Shared.Morgue;
+
+[Serializable, NetSerializable]
+public enum MorgueVisuals
 {
-    [Serializable, NetSerializable]
-    public enum MorgueVisuals
-    {
-        Open,
-        HasContents,
-        HasMob,
-        HasSoul,
-    }
+    Open,
+    HasContents,
```

Comment:
Open & HasContents should probably be some generic storage visuals & not  morgue specific